edgexfoundry/edgex-compose
# build: Update Device Camera version to 2.0.1

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have opened a PR for the related docs change (if not, why?) **Not applicable**



## Testing Instructions
From compose builder 
1. run `make run no-secty ds-camera`
2. Verify `edgexfoundry/device-camera:2.0.1` is pulled and is running.
